tst/regression/test_suites/restart/restart.py

```diff
@@ -38,8 +38,7 @@ def Prepare(self, parameters, step):
             parameters.driver_cmd_line_args = [
                 '-r',
                 'gold.out0.00001.rhdf',
-                'parthenon/job/problem_id=silver',
-                'parthenon/mesh/nx1=%d' % (64 * parameters.num_ranks)
+                'parthenon/job/problem_id=silver'
```

Comment:
Does this mean the restart regression test can only be used with a singe gpu/proc?

Reply:
Nevermind, I just answered my own question.